regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -111,9 +112,9 @@ var CommentView = Backbone.View.extend({
         label = [label, $sectionHeader.text().split('. ').slice(1)].join('. ');
         $sectionHeader.remove();
       }
-      this.$header.append('<a href="' + href + '">' + label + '</a>');
+      this.$headerLink.html('<a href="' + href + '">' + label + '</a>');
```

Comment:
No need to hold this PR back, but would it make sense to have the `a` present in the markup and set its attributes instead? We'd avoid generating markup in the JS.